### PR TITLE
Make icons compatible with flatpak

### PR DESCRIPTION
**Pull request by illwieckz**
Flatpak requires the icon names to be named like the flatpak.

### CODE REVIEW

**Comment by illwieckz** [comment]
For information, here is a work-in-progress flatpak recipe based on this commit:

- https://github.com/illwieckz/flathub/tree/unvtray

Unfortunately the icon in tray icon browser looks bad and right click doesn't open any menu… Left clicks works. But this is a flatpak issue and should not prevent this PR to be merged.

File: make_icons.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 import unvtray
 from unvtray.tray import make_icon
 
-APP_NAME = unvtray.__name__  # pyright: ignore
+APP_NAME = "net.unvanquished." + unvtray.__name__  # pyright: ignore
```
Comment (Viech) [inline make_icons.py:25]: For consistency I'd prefer `f"net.unvanquished.{unvtray.__name__}"` or simply `"net.unvanquished.unvtray"` (then without the `pyright` hint).

**Review by mjstahlberg — CHANGES_REQUESTED** [review]
There is also a link to an icon in `README.md`:

```html
README.md:8:  <img src="package/icons/128x128/unvtray.png" />
```

With the changes, you can go ahaead and push to `main`; you can also tag `v0.2.2` if that's necessary.

**Comment by illwieckz** [comment]
I may have other minor changes to do so I see no urge for a tag. The flatpak itself doesn't require a tag and until it is ready using a commit id is as good.

**Pull request merged**